### PR TITLE
add basic doc...

### PR DESCRIPTION
**Pull request by fieker**

### CODE REVIEW

File: experimental/GaloisGrp/Solve.jl [hunk 1]
```diff
@@ -248,23 +264,50 @@ function _fixed_field(C::GaloisCtx, s::Vector{PermGroup}; invar=nothing, max_pre
   return k
 end
 
-function fixed_field(C::GaloisCtx, s::Vector{PermGroup}; invar=nothing, max_prec::Int = typemax(Int))
-  return number_field(_fixed_field(C, s, invar = invar, max_prec = max_prec))
+@doc Markdown.doc"""
+    fixed_field(C::GaloisCtx, s::Vector{PermGroup})
+
+Given a descending chain if subgroups, each being maximal in the previous
```
Comment (fingolfin) [inline experimental/GaloisGrp/Solve.jl]:
```suggestion
Given a descending chain of subgroups, each being maximal in the previous
```

File: experimental/GaloisGrp/Solve.jl [hunk 2]
```diff
@@ -248,23 +264,50 @@ function _fixed_field(C::GaloisCtx, s::Vector{PermGroup}; invar=nothing, max_pre
...
+one, compute the corresponding subfields as a tower.
+
+# EXAMPLES
```
Comment (fingolfin) [inline experimental/GaloisGrp/Solve.jl]:
```suggestion
# Examples
```

File: experimental/GaloisGrp/Solve.jl [hunk 3]
```diff
@@ -297,7 +340,14 @@ function recognise(C::GaloisCtx, S::SubField, J::Vector{<:SLPoly})
   return D    
 end
 
+"""
+For a cyclic extension K/k with Automorphism group generated by aut and
+a corresponding n-th root of 1, find an isomorphic radical extension
```
Comment (fingolfin) [inline experimental/GaloisGrp/Solve.jl]: Maybe this?
```suggestion
a corresponding primitive n-th root of 1, find an isomorphic radical extension
```

File: experimental/GaloisGrp/Solve.jl [hunk 4]
```diff
@@ -316,16 +366,60 @@ function as_radical_extension(K::NumField, aut::Map, zeta::NumFieldElem)
...
+  return L, hom(L, K, r, check = CHECK)
 end
 
 function Oscar.solve(f::fmpq_poly; max_prec::Int=typemax(Int))
   return solve(numerator(f), max_prec = max_prec)
 end
 
+@doc Markdown.doc"""
+    Oscar.solve(f::fmpz_poly; max_prec::Int=typemax(Int))
+    Oscar.solve(f::fmpq_poly; max_prec::Int=typemax(Int))
```
Comment (fingolfin) [inline experimental/GaloisGrp/Solve.jl:381]: Isn't this exported?
```suggestion
    solve(f::fmpz_poly; max_prec::Int=typemax(Int))
    solve(f::fmpq_poly; max_prec::Int=typemax(Int))
```

File: experimental/GaloisGrp/Solve.jl [hunk 5]
```diff
@@ -316,16 +366,60 @@ function as_radical_extension(K::NumField, aut::Map, zeta::NumFieldElem)
...
+
+Compute a presentation of the roots of `f` in a radical tower.
+The neccessary roots-of-1 are not themselves computed as radicals.
+
+See also: `galois_group`
```
Comment (fingolfin) [inline experimental/GaloisGrp/Solve.jl]:
```suggestion
See also [`galois_group`](@ref).
```

File: experimental/GaloisGrp/Solve.jl [hunk 6]
```diff
@@ -316,16 +366,60 @@ function as_radical_extension(K::NumField, aut::Map, zeta::NumFieldElem)
...
 function Oscar.solve(f::fmpq_poly; max_prec::Int=typemax(Int))
   return solve(numerator(f), max_prec = max_prec)
 end
 
+@doc Markdown.doc"""
+    Oscar.solve(f::fmpz_poly; max_prec::Int=typemax(Int))
+    Oscar.solve(f::fmpq_poly; max_prec::Int=typemax(Int))
+
+Compute a presentation of the roots of `f` in a radical tower.
+The neccessary roots-of-1 are not themselves computed as radicals.
```
Comment (fingolfin) [inline experimental/GaloisGrp/Solve.jl]:
```suggestion
The necessary roots of unity are not themselves computed as radicals.
```

File: experimental/GaloisGrp/Solve.jl [hunk 7]
```diff
@@ -316,16 +366,60 @@ function as_radical_extension(K::NumField, aut::Map, zeta::NumFieldElem)
...
+
+See also: `galois_group`
+
+# VERBOSE
+Supports `set_verbose_level(:SolveRadical, i)` to obtain information.
+
+
+# EXAMPLES
```
Comment (fingolfin) [inline experimental/GaloisGrp/Solve.jl]:
```suggestion
# Examples
```

File: experimental/GaloisGrp/Solve.jl (hunk 4 above)
Comment (fieker) [reply experimental/GaloisGrp/Solve.jl:381]: no can do: solve is not imported...

**Pull request merged**